src/odoc/etc/odoc.css

```diff
@@ -676,9 +676,12 @@ td.def-doc *:first-child {
 /* Mobile adjustements. */
 
 @media only screen and (max-width: 95ex) {
+  body.odoc {
+    margin: 2em;
+  }
   .odoc-content {
-    margin: auto;
-    padding: 2em;
+    margin: 0;
+    padding: 0;
```

Comment:
These are divs and it's not overriding anything, so you can drop the entire rule.